genapi/genapi.go

```diff
@@ -236,6 +236,10 @@ type TLSInfo struct {
 	// SessionTicketKey leaks, previously recorded and future TLS connections
 	// using that key are compromised.
 	SessionTicketKey [32]byte
+
+	// If set then this config will be used for all tls listeners, regardless of
+	// all other factors.
+	ForceConfig *tls.Config
```

Comment:
Also add a comment that if this is set, you need to call `BuildNameToCertificate` otherwise it won't work at all.